ci/jenkins/jobs/projects-lab.yaml

```diff
@@ -872,16 +872,16 @@
             wrappers: [ ]
             publishers: [ ]
       - '{name}-{test_name}-for-pull-request':
-            test_name: windows-containerd-conformance
+            test_name: windows-containerd-with-ovs-conformance
```

Comment:
@NamanAg30 , 
To shorten trigger phrase name, can prefer to do away with string like containerd in trigger phrases and everywhere else and can keep only 'ovs' instead of 'with-ovs' in the string until docker based job is removed

Reply:
we have both containerd and containerd-with-ovs jobs for e2e tests , that is why we have used this naming convention.... and therefore to maintain consistency we have used the same in case of conformance and network policy tests

Reply:
changes made